src/funnels/Depositor.sol

```diff
@@ -74,16 +74,16 @@ contract Depositor {
     struct PairLimit {
         uint64   hop; // Cooldown one has to wait between changes to the liquidity of a (gem0, gem1) pool
         uint64   zzz; // Timestamp of the last liquidity change for a (gem0, gem1) pool
-        uint128 cap0; // Maximum amt of gem0 that can be added as liquidity each hop for a (gem0, gem1) pool
-        uint128 cap1; // Maximum amt of gem1 that can be added as liquidity each hop for a (gem0, gem1) pool
+        uint128 cap0; // Maximum amount of gem0 that can be added as liquidity each hop for a (gem0, gem1) pool
+        uint128 cap1; // Maximum amount of gem1 that can be added as liquidity each hop for a (gem0, gem1) pool
     }
 
     event Rely(address indexed usr);
     event Deny(address indexed usr);
     event SetLimits(address indexed gem0, address indexed gem1, uint64 hop, uint128 cap0, uint128 cap1);
-    event Deposit(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1);
-    event Withdraw(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1, uint256 collected0, uint256 collected1);
-    event Collect(address indexed sender, address indexed gem0, address indexed gem1, uint256 collected0, uint256 collected1);
+    event Deposit (address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1);
+    event Withdraw(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1, uint256 fees0, uint256 fees1);
+    event Collect (address indexed sender, address indexed gem0, address indexed gem1, uint256 fees0, uint256 fees1);
```

Comment:
I think we should remove the extra spaces excepting we really want to start aligning all the events.